controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -200,6 +208,13 @@ func (r *StorageClassClaimReconciler) reconcileConsumerPhases() (reconcile.Resul
 				return reconcile.Result{}, fmt.Errorf("failed to unmarshal StorageClassClaim configuration response: %v", err)
 			}
 
+			// Add received resources to the StorageClassClaim Status.
+			r.storageClassClaim.Status.CephResources = append(r.storageClassClaim.Status.CephResources, &ocsv1alpha1.CephResourcesSpec{
+				Kind: resource.Kind,
+				Name: resource.Name,
+			})
```

Comment:
@umangachapagain @Madhu-1 Need some clarification for the `CephClients` (need this CR for the client name to map it to the resource) and `Phase` (no phase on secret or storage class) fields here.

Reply:
Please see my comment regarding the removal of the ceph resource from the status section of the consumer storage class claim here: https://github.com/red-hat-storage/ocs-operator/pull/1608#discussion_r859140876